### PR TITLE
add _TZE200_0u3bj3rc tuya human presence sensor

### PR DESCRIPTION
**Pull request by Burki24**

### CODE REVIEW

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1525/

File: devices/tuya.js [hunk 1]
```diff
@@ -3073,4 +3073,21 @@ module.exports = [
             await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg', 'msTemperatureMeasurement', 'msRelativeHumidity']);
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_0u3bj3rc'}],
+        model: 'Human_presence_sensor',
```
Comment (Koenkk) [inline devices/tuya.js:3078]: Is there any other model number written on the device? If not please use `TS0601_human_presence_sensor`.

Comment (Burki24) [reply devices/tuya.js]: finished, thanks

**Comment by Koenkk** [comment]
Thanks!

**Comment by Burki24** [comment]
@Koenkk 

thanks for putting tz and fz into tzlocal and fzlocal. Is that way of adding  for all devices in the future? Then I will update only the device-scripts from now on and will not do changes in fromzigbee and tozigbee.

**Comment by Koenkk** [comment]
@Burki24 for specific converters yes, otherwise `fromZigbee.js`/`toZigbee.js` are getting too big.

**Pull request merged**